platform-tests/acceptance/aws_endpoint_check_test.go

```diff
@@ -56,7 +30,8 @@ var _ = Describe("AWS Endpoint Check", Ordered, func() {
 		)
 
 		BeforeAll(func() {
-			region, err = GetAWSRegion()
+			region = os.Getenv("AWS_REGION")
```

Comment:
I was wondering if we had that env var, but I assumed you'd thought of it already

Reply:
Yeah, probably should have gone there first.